### DISCUSSION TITLE
Change and Migrate Storage Location

### DISCUSSION THREAD

**Discussion by itgav** [erigontech/erigon#4759] [Q&A] [answered]
Hi,
I'm still trying to learn computers so excuse me if any of this is elementary. So, I built a new computer and installed a 1TB SSD that has my OS (windows... ik) and all my other programs on it. I began downloading Erigon on that 1TB drive until my 4TB drive came in the mail. My goal is to either copy the Erigon data on the 1TB to the larger drive and then continue the download. Or, start the download over and store it in the larger drive.

I redownloaded the Erigon Github and ran the below code: --> A: is my new 4TB storage
./build/bin/erigon --datadir=A:\Erigon_Repo\Erigon_Data 

Erigon will continue building, but the data is still being stored on my 1TB drive (C: drive).

Does anyone know how to solve this? Is this where a symlink is needed?
Any help is appreciated, thank you.

**Comment by AskAlexSharov** [comment, marked as answer]
1. Can copy data directory to bigger drive: from `C:\Erigon_Repo\Erigon_Data` to `A:\Erigon_Repo\Erigon_Data`. Example: https://stackoverflow.com/questions/7170683/copy-all-files-and-folders-from-one-drive-to-another-drive-using-dos-command-pr
2. If you don't need to keep all Ethereum history (you don't plan to do history analytics and other historical requests). If having last 90K blocks of history is enough: can add flag `--prune=hrtc` (but need start sync from scratch to add this flag, cant change it after first erigon start). It will reduce datadir size.

**Reply by itgav** [reply]
1. Thank you for the link.

It seems that my confusion was from the intermittent storage fluctuations on my C: drive vs the gradual addition of storage to the database on my A: drive.

Now, I copied the data from the C: drive to the A: drive, and then deleted the data that was copied from the C: drive to free up space. Then, I ran the Erigon client with --datadir set to my A: drive. A: drive is where the database is stored; C: drive is where the GitHub repository is stored, OS, and all my other files. Executed this process in the L30 min so not a lot of change in storage yet. But, from what I can tell the database is being correctly stored to the A: drive. As a part of this process, the C: drive is having storage fluctuate so I would imagine there's a bin on that drive that holds intermittent copies of the data until it is officially written to the database on the A: drive, then they're deleted from the C: drive. Not sure if this is because my C: drive has the repository on it or if it's a function of Windows/all my other files being on the C: drive.

The original confusion I had stemmed from my C: drive being full (didn't delete the copied database files originally) when I first copied the database over to my A: drive. I would see the intermittent storage on the C: drive would fluctuate, the program would stop due to full storage, and I assumed the data was still being saved to the C: drive which I'm learning is false.

The intermittent storage on my C: drive has been fluctuating by 0-22GB but ultimately reverts to 0 additional GBs stored. This is all in the process of adding 2 additional GBs of storage to the database on my A: drive.

All is well. Go Erigon and thank you @AskAlexSharov.
